index.js

```diff
@@ -115,6 +115,7 @@ function ParseServer(args) {
   router.merge(require('./installations'));
   router.merge(require('./functions'));
   router.merge(require('./schemas'));
+  router.merge(require('./global_config'));
```

Comment:
For compatibility with parse.com I think we need to name this route `config`. We'll probably also want to hide this behind an experimental flag until we have PUT and DELETE as well and have checked to make sure the parse dashboard is fully compatible (we didn't do this for the schemas API, but I think we should have). We don't have a fully fleshed out mechanism for enabling experimental features yet, but my suggestion would be to branch based on process.env.PARSE_EXPERIMENTAL_CONFIG_ENABLED or something like that. Totally up to you though :)